README.md

```diff
@@ -33,7 +33,7 @@
 
 # 📖 [OOP Ruby School Library ] <a name="about-project"></a>
 
-> ## **[OOP Ruby School Library]** :- Simple ruby program that build a school library using the oriented object approach. The library keeps records of all the books and the students and teachers who borrowed them
+> ## **[OOP Ruby School Library]** :- Simple ruby program that build a school library using the oriented object approach. The library keeps records of all the books and the students and teachers who borrowed them and besides it was added Ruby decorators that help to add behaviours to objects without affecting other objects of the same class.
 
 ## 🛠 Built With <a name="built-with"></a>
 
```

Comment:
This change is being added here because you did not make changes to the person file.

- [ ] The Person class should inherit from the Nameable class

- [ ] The Person class should have a corresponding `correct_name` method

  ```
  require_relative './nameable'

  class Person < Nameable
    ...
    def correct_name
      @name
    end
    ...
  end
  ```